suites/compaction/deployments/rbd-mirror-deploy-and-configure.yaml

```diff
@@ -0,0 +1,205 @@
+#===============================================================================================
+# Tier-level: 1
```

Comment:
pls remove all these that are not applicable here.

Reply:
removed all comments